sample-code/src/orchestration.ts

```diff
@@ -656,10 +627,11 @@ export async function orchestrationMessageHistoryWithToolCalling(): Promise<Orch
     allMessages.push(message);
   }
 
-  return orchestrationClient(
-    [{ role: 'user', content: 'What is the corresponding roman numeral?' }],
-    addNumbersTool
-  ).chatCompletion({
+  // Call the model with a new message and the message history
+  return orchestrationClient.chatCompletion({
+    messages: [
+      { role: 'user', content: 'What is the corresponding roman numeral?' }
+    ],
     messagesHistory: allMessages
```

Comment:
[q] Maybe I missed some of the discussion, but are we separating now messages from history?

Reply:
I thought we deprecate `messagesHistory`?